src/types/test/geography.spec.ts

```diff
@@ -0,0 +1,487 @@
+import { Geography } from "../geography";
```

Comment:
There's some repeated fixture data in the test cases. It could be a good idea to declare those objects outside the test cases.

These two for example:

const EMPTY_OBJECT = { geo: "", longitude: 0, latitude: 0, featureType: "" };
const ZERO_POINT =   { geo: "0,0", longitude: 0, latitude: 0, featureType: "POINT" };